### DISCUSSION TITLE
2 node HA Cluster for veeam offsite backup with same external address for bucket)

### DISCUSSION THREAD

**Discussion by fsennj** [minio/minio#14007] [Q&A] [answered]
Hello All, for our veeam offsite backup i wanted to use 2 nodes each with 8 disks in EC:2 mode (so ~ Raid 6) but i would like to have both nodes under the same external domain reachable so replication is not really usefull here.

is it possible to have the 2 nodes in a mirrored cluster (i setup with distributed minio command minio server xx-{1...2}.xxxx.at\data{1...8})

so N1 (8Disks EC:2) mirrored to N2 (8 Disks EC:2) or is it only available with 4 nodes minimum?

thank you

**Comment by harshavardhana** [comment, marked as answer]
> Hello All, for our veeam offsite backup i wanted to use 2 nodes each with 8 disks in EC:2 mode (so ~ Raid 6) but i would like to have both nodes under the same external domain reachable so replication is not really usefull here.
> 
> is it possible to have the 2 nodes in a mirrored cluster (i setup with distributed minio command minio server xx-{1...2}.xxxx.at\data{1...8})
> 
> so N1 (8Disks EC:2) mirrored to N2 (8 Disks EC:2) or is it only available with 4 nodes minimum?
> 
> thank you

2 nodes is not a supported configuration for production. You need a minimum of 3 nodes for HA.

**Reply by fsennj** [reply]
Thank you i have done now with Replication
